source/guide_tarsnap.rst

```diff
@@ -0,0 +1,232 @@
+ [isabell@stardust ~]$ gpg --list-packets tarsnap-signing-key-2021.asc | grep signature
+ [isabell@stardust ~]$ gpg --import tarsnap-signing-key-2021.asc
+ [isabell@stardust ~]$ rm tarsnap-signing-key-2021.asc
+ [isabell@stardust ~]$
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ wget https://www.tarsnap.com/download/tarsnap-autoconf-1.0.39.tgz
+ [isabell@stardust ~]$ wget https://www.tarsnap.com/download/tarsnap-sigs-1.0.39.asc
+ [isabell@stardust ~]$ gpg --decrypt tarsnap-sigs-1.0.39.asc
+ [isabell@stardust ~]$ shasum -a 256 tarsnap-autoconf-1.0.39.tgz
+ [isabell@stardust ~]$ rm tarsnap-sigs-1.0.39.asc
+ [isabell@stardust ~]$
+
+
+Extract and compile the code
+----------------------------
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ tar -xzf tarsnap-autoconf-1.0.39.tgz
+ [isabell@stardust ~]$ cd tarsnap-autoconf-1.0.39/
+ [isabell@stardust tarsnap-autoconf-1.0.39]$ ./configure --prefix=/home/$USER
+ [isabell@stardust tarsnap-autoconf-1.0.39]$ make
+ [isabell@stardust tarsnap-autoconf-1.0.39]$ make install
+ [isabell@stardust tarsnap-autoconf-1.0.39]$ cd ..
+ [isabell@stardust ~]$ rm -rf tarsnap-autoconf-1.0.39
+ [isabell@stardust ~]$ rm tarsnap-autoconf-1.0.39.tgz
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Create a working directory
+--------------------------
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ mkdir /home/$USER/tarsnap
+ [isabell@stardust ~]$
+ 
+
+Create a user account and preload money to it
+---------------------------------------------
+
+Go to https://www.tarsnap.com/register.cgi and register an account. After that load some money to it. At least $5.
+
+
+Create some keyfiles
+--------------------
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ tarsnap-keygen \
+							--keyfile /home/$USER/tarsnap/tarsnap.key \
+							--user your_registered_email_from_the_account_registration \
+							--machine some_nifty_name
+							--passphrased
+ [isabell@stardust ~]$
+
+As this key has all rights to manage your backups, you have to create a key with write-only access to do backups automatically per cronjob.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ tarsnap-keymgmt \
+							--outkeyfile /home/$USER/tarsnap/tarsnapwrite.key \
+							-w /home/$USER/tarsnap/tarsnap.key``
+ [isabell@stardust ~]$
+
+.. warning:: Please copy your keyfiles to a safe place! Without the keys you cannot access your backups anymore.
+
+
+Set up the config file
+----------------------
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ cp /home/$USER/etc/tarsnap.conf.sample /home/$USER/etc/tarsnap.conf
+ [isabell@stardust ~]$
+
+Now edit ``/home/$USER/etc/tarsnap.conf`` with the editor of your choice. The config file is already commented by the author.
+Make sure to use your write-only keyfile under keyfile in the config file.
```

Comment:
Please include an example in this step. You can check the [mumble guide](https://lab.uberspace.de/guide_mumble.html#configuration) for some hints. Something like this:
```suggestion
Make sure to use your write-only keyfile under keyfile in the config file:

.. code-block::
 :emphasize-lines: 7

 ### Recommended options

 # Tarsnap cache directory
 cachedir /usr/local/tarsnap-cache

 # Tarsnap key file
 keyfile /home/tsnap/tarsnap/tarsnap.key
```